react/features/base/participants/components/styles.js

```diff
@@ -1,9 +1,32 @@
-import { createStyleSheet } from '../../styles';
+import { BoxModel, ColorPalette, createStyleSheet } from '../../styles';
 
 /**
  * The styles of the feature base/participants.
  */
 export default createStyleSheet({
+    /**
+     * Style for the text rendered when there is a connectivity problem.
+     */
+    interruptedText: {
+        color: ColorPalette.white,
+        fontSize: 12,
+        marginVertical: BoxModel.margin,
+        marginHorizontal: BoxModel.margin / 2,
```

Comment:
Will that be consistent with other places in our app ? Do we usually use this size of margin, between the left and right side of the screen ? Or does it just looks better this way instead of using full BoxModel.margin ?

Reply:
Note that this is the style for the text. The idea is to give it some space within the container, so it doesn't look that close to the edges. I don't think we have any other component similar to this.